lib/person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_reader :name,
+              :interests,
+              :supplies
+  def initialize(info)
+    @name = info[:name]
+    @interests = info[:interests]
+    @supplies = {}
+  end
+
+  def add_supply(supply, amount)
+    if @supplies.keys.include?(supply) == false
+      @supplies[supply] = amount
+    else
+      @supplies[supply] += amount
```

Comment:
what if you just called line 15 and got rid of your `if/else`? Would it still work?